dvc/progress.py

```diff
@@ -1,8 +1,8 @@
 """Manages progress bars for dvc repo."""
 from __future__ import print_function
 import logging
-from tqdm import tqdm
 from concurrent.futures import ThreadPoolExecutor
+from tqdm import tqdm
```

Comment:
To separate standard imports from 3-rd party we need to add a newline here
```suggestion

from tqdm import tqdm
```